systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -93,6 +98,7 @@
     // Image pull policy variables
     public static final String COMPONENTS_IMAGE_PULL_POLICY = System.getenv().getOrDefault(COMPONENTS_IMAGE_PULL_POLICY_ENV, COMPONENTS_IMAGE_PULL_POLICY_ENV_DEFAULT);
     public static final String OPERATOR_IMAGE_PULL_POLICY = System.getenv().getOrDefault(OPERATOR_IMAGE_PULL_POLICY_ENV, OPERATOR_IMAGE_PULL_POLICY_ENV_DEFAULT);
+    public static final String ALLOW_NETWORK_POLICIES = System.getenv().getOrDefault(ALLOW_NETWORK_POLICIES_ENV, ALLOW_NETWORK_POLICIES_DEFAULT);
```

Comment:
If I read the code right, this does not allow or disable the network policies. Thsi is rather a swicth between default-to-allow and default-to-deny network policies. So I think we should change the name to make it more clear.